src/shared/proto/tbots_frankie_status.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto2";
+
+message Tbots_EncoderData {
```

Comment:
UberNit: Can we remove the mix of snake_case and just use CamelCase for proto

so `Tbots_EncoderData` -> `TbotsEncoderData`

Reply:
Perhaps we should define a style guide for our own Protos now that we are writing our own (will be the same deal for primitive messages)

Reply:
I tried to imitate the the style of SSL_WrapperPacket naming, but we can define our own style if we'd like

Reply:
I don't have a strong preference as long as we're consistent. Perhaps easiest to just follow Google's? https://developers.google.com/protocol-buffers/docs/style

Reply:
I would strongly support just following google's, it will prevent debate and make our proto format more similar to others.

Reply:
Agreed

Reply:
@jonathanlew mind adding that to the style guide docs in this PR then?

Reply:
done